src/triton_bringup/launch/teleop_sim_bringup_launch.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+from launch import LaunchDescription
+from launch_ros.actions import Node
+from launch.actions import IncludeLaunchDescription, TimerAction
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from ament_index_python.packages import get_package_share_directory
+
+
+def generate_launch_description():
+    ld = LaunchDescription()
+
+    pkg_share = get_package_share_directory('triton_gazebo')
+    urdf_file =  os.path.join(pkg_share, 'gazebo', 'models', 'cube_auv', 'model.urdf')
```

Comment:
Should we add a parameter here instead of launching the cube_auv model directly?

Reply:
I think you can just replace this with the triton_auv model on your branch, if we have that model, can't see why we would use the cube_auv one, I just used it because it's easier to set things up with

Reply:
Okay yeah fair enough, thanks